newsroom/news_api/formatters/service.py

```diff
@@ -33,7 +35,10 @@ def get_version(self, id, version, formatter_name):
             item['_id'] = item[id_field]
         else:
             item = get_resource_service('items').find_one(req=None, _id=id)
-        if not item:
+            if not item:
+                abort(404)
+        # Ensure that the item has not expired
+        if utcnow() - timedelta(days=app.config.get('MAX_EXPIRY_QUERY_LIMIT')) > item.get('versioncreated', utcnow()):
```

Comment:
There is a specific config for the News API `news_api_time_limit_days` (https://github.com/superdesk/newsroom/blob/master/newsroom/news_api/__init__.py#L10), and a utility method for retrieving this setting.

```python
from newsroom.settings import get_setting
get_setting('news_api_time_limit_days')
```